app/helpers/comments_helper.rb

```diff
@@ -31,6 +31,16 @@ def should_be_hidden?(comment, root_comment)
     comment.hidden_by_commentable_user && comment != root_comment
   end
 
+  def like_button_text(comment)
+    if comment.public_reactions_count.zero?
+      "Like"
+    elsif comment.public_reactions_count == 1
+      "&nbsp;like"
+    else
+      "&nbsp;likes"
+    end
+  end
```

Comment:
**nitpick (non-blocking):** maybe we could use a case statement here if we feel it's easier to read:

```ruby
case comment.public_reactions_count
when 0
  "Like"
when 1
  "&nbsp;like"
else
  "&nbsp;likes"
end
```

Or if we want to import `pluralize` we can do something like:
```ruby
return "Like" if comment.public_reactions_count.zero?

"&nbsp;#{pluralize(comment.public_reactions_count, "like")}"

```